test_prod/razorpay.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Razorpay = require("../dist/razorpay");
+let request = require('request-promise');
+
+class RazorpayTest extends Razorpay {
+  constructor(options) {
+    super(options)
+    this.api.rq = request.defaults({
```

Comment:
Can we pass `hostUrl` as options in constructor? 
By default, it would be `api.razorpay.com` else it can be `https://api-web.dev.razorpay.in`

Reply:
added hostUrl in options

Reply:
I didn't mean adding it in RazorpayTest but in Razorpay class. In that case, we don't need RazorpayTest itself. We can directly use Razorpay by passing hostUrl. 
Sorry for this confusion.

Reply:
Just realised that adding it in class Razorpay would have other problems since that is a public API. Let's keep this as such for now.